src/app/services/article.service.ts

```diff
@@ -13,11 +13,14 @@ export class ArticleService {
     "stories",
     "church-construction-complete",
     "is-peter-rock",
-    "why-not-catholic"
+    "why-not-catholic",
+    "church-construction-complete",
+    "is-peter-rock",
+    "why-not-catholic",
   ];
-  public defaultImageURI = "https://t4.ftcdn.net/jpg/02/97/78/03/360_F_297780357_pK8VCA7wctbTFusAGiCfcoxbJLRwC9Bs.jpg";
-  public NEW_LABEL:string = "[ new ]";
-  public BASE_FIRESTORE: string = "https://firestore.googleapis.com/v1";
+  public readonly defaultImageURI = "https://t4.ftcdn.net/jpg/02/97/78/03/360_F_297780357_pK8VCA7wctbTFusAGiCfcoxbJLRwC9Bs.jpg";
+  public readonly NEW_LABEL: string = "[ new ]";
```

Comment:
for consistency with my statement about having fake 'article", we should probably drop this as well and just set documentId and name as null.